tests/test_exposure.py

```diff
@@ -777,6 +777,88 @@ def testGetCutout(self):
                     with self.assertRaises(pexExcept.InvalidParameterError, msg=msg):
                         self.smallExposure.getCutout(cutoutCenter, cutoutSize)
 
+    def testGetConvexPolygon(self):
+        """Test the convex polygon."""
+        # Check that we do not have a convex polygon for the plain exposure.
+        self.assertIsNone(self.exposureMiOnly.convex_polygon)
+
+        # Check that all the points in the bounding box are in the polygon
+        bbox = self.exposureMiWcs.getBBox()
+        # Grow by the default padding.
+        bbox.grow(10)
+        x, y = np.meshgrid(np.arange(bbox.getBeginX(), bbox.getEndX()),
+                           np.arange(bbox.getBeginY(), bbox.getEndY()))
+        wcs = self.exposureMiWcs.wcs
+        ra, dec = wcs.pixelToSkyArray(x.ravel().astype(np.float64),
```

Comment:
Why astype here?

Reply:
I suppose this should be in the `np.arange`.  Done.